src/main/java/org/rust/lang/core/psi/RustTokenElementTypes.java

```diff
@@ -54,7 +54,7 @@
     RustTokenType SUPER = new RustKeywordTokenType("super");
     RustTokenType TRAIT = new RustKeywordTokenType("trait");
     RustTokenType TRUE = new RustKeywordTokenType("true");
-    RustTokenType TYPE = new RustKeywordTokenType("type");
+    RustTokenType TYPE_KW = new RustKeywordTokenType("type");
```

Comment:
What do you think about prefixing all keyword token types with `KW_`, e.g. `KW_TYPE`, `KW_UNSIZED`? This will save us from getting imho awful name collisions like this one.

The same idea might be applied to operators - `OP_xxx`. Thought, unfortunately, this would be breaking change.